### PR TITLE
docs(grid): document grid and provide examples - INNO-1616

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 240e516b2e94dfa9d74136122c48ae6d68dd93db

https://deploy-preview-1189--europa-component-library.netlify.com

File: src/website/src/pages/ec/layout/grid/index.mdx [hunk 1]
```diff
@@ -3,29 +3,98 @@ title: Grid
...
+  selectedStory="3-3-6"
+  showFrame
+>
+  <Example336 />
+</Playground>
 
-![An image of a grid](https://inno-ecl.s3.amazonaws.com/media/images/EC/Grid/Grid.svg)
+## Grid offset
+
+When using the ECL grid, there is the ability to offset columns. You can offset columns from left to right by invoking the `.ecl-offset-*` classes (where \* = the number of columns). These classes simply increases the left margin by \* number of columns.
```
Comment (planctus) [inline src/website/src/pages/ec/layout/grid/index.mdx]: Well, should be "These classes simply increase"..

**Review by planctus — CHANGES_REQUESTED** [review]
Well, as a request for changes is pretty awkward, i only found one "typo". For the rest it is working as expected for me, and the content is perfectly matching the wiki page, although still not final.

File: src/website/src/pages/ec/layout/grid/index.mdx (hunk 1 above)
Comment (yhuard) [reply src/website/src/pages/ec/layout/grid/index.mdx]: good catch :) I've fixed it in my last commit

**Review by planctus — APPROVED** [review]

**Pull request merged**